### ISSUE TITLE
Implement SGD Optimizer with tensorflow and pytorch

### ISSUE THREAD

**Issue by pavanreddyml** [pavanreddyml/adversarial-lab#8]
Implement a Stochastic Gradient Descent(SGD) with tf or torch similar to "adversarial_lab/core/optimizers/adam.py"
